src/forms/GoogleMapsField.php

```diff
@@ -48,6 +52,7 @@ class GoogleMapsField extends TextField
     public function __construct($name, $title = null, $value = '', $maxLength = null, Form $form = null)
     {
         $this->setAttribute('data-mapsfield', 'mapsfield');
+        $this->setCustomisationsVarName($name . 'Customisations');
```

Comment:
It should be a single instance with separate options in an array, not a set of separates. Therefore, statics should be used.